src/tour.ts

```diff
@@ -89,41 +74,41 @@ tour.addStep({
 
 tour.addStep({
   attachTo: {
-    element: '[data-tour=csv]',
+    element: '[data-tour=navmode]',
     on: 'auto',
   },
-  title: 'Load CSV File',
-  text: `EpiVis supports loading CSV files to explore custom datasets.`,
+  title: 'Toggle between Navigation Modes',
+  text: `EpiVis supports four navigation modes. Autofit ensures that all enabled datasets are fully displayed on the chart; while Pan, Crop, and Zoom can be used to manipulate the view.`,
```

Comment:
```suggestion
  title: 'Toggle between Navigation Modes',
  text: `EpiVis supports four navigation modes: Autofit (the default) ensures that all enabled datasets are fully displayed within the chart, while Pan, Crop, and Zoom can be used to manipulate the view.`,
```

Also, can we move the 3 mouse panning/cropping/zooming tour steps between the "Main Chart" and "Chart Menu" steps?  That will keep the chart-menu-specific stuff together, which i think is clearer.